filters/tracing/tag.go

```diff
@@ -46,23 +80,43 @@ func (s *tagSpec) CreateFilter(args []interface{}) (filters.Filter, error) {
 		return nil, filters.ErrInvalidFilterParameters
 	}
 
-	return &tagFilter{
-		tagFromResponse: s.typ == filters.TracingTagFromResponseName,
-
+	f := &tagFilter{
+		typ:      typ,
 		tagName:  tagName,
 		tagValue: eskip.NewTemplate(tagValue),
-	}, nil
+	}
+
+	if len(args) == 4 {
+		minValue, ok := args[2].(int)
```

Comment:
The actual type from eskip will be float64

Reply:
thanks fixed and test adapted to test it